integrations/functions.php

```diff
@@ -10,42 +10,71 @@
  */
 
 /**
- * Registers all default service workers.
+ * Registers service worker integrations.
+ *
+ * These integrations are separate from the service worker core and need to be opted in via theme support.
+ * To enable all integrations, use the following:
+ *
+ *     add_theme_support( 'service_worker', true );
+ *
+ * Alternatively, you can also pass an array of integration slugs as keys and a boolean indicating that
+ * integration's status as values, for example:
+ *
+ *     add_theme_support(
+ *         'service_worker',
+ *         array(
+ *             'wp-custom-logo'   => true,
+ *             'wp-custom-header' => true,
```

Comment:
Since they all start with `wp-` should this part be omitted?

Reply:
I think they should remain as it clarifies these are "WordPress default features" (as in the PWA being a feature plugin). Other plugins could add their own and integrate with this, then use another prefix.